cypress/integration/user_profile/SocialMedia.feature

```diff
@@ -27,3 +27,16 @@ Feature: List Social Media Accounts
     Given I have added a social media link
     When I delete a social media link
     Then it gets deleted successfully
+
+  Scenario: Editing Social Media
+    Given I am on the "settings" page
+    And I click on the "Social media" link
+    Then I should be on the "/settings/my-social-media" page
+    Given I have added a social media link
+    When I start editing a social media link
+    Then I can cancel editing
+    When I start editing a social media link
+    And I edit and save the link
+    Then it gets saved successfully
+    And the new url is displayed
+    But the old url is not displayed
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-18T07:25:03Z" title="Thursday, July 18th 2019, 9:25:03 am +02:00">Jul 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -27,3 +27,16 @@ Feature: List Social Media Accounts
     Given I have added a social media link
     When I delete a social media link
     Then it gets deleted successfully
+
+  Scenario: Editing Social Media
+    Given I am on the "settings" page
+    And I click on the "Social media" link
+    Then I should be on the "/settings/my-social-media" page
+    Given I have added a social media link
+    When I start editing a social media link
+    Then I can cancel editing
+    When I start editing a social media link
+    And I edit and save the link
+    Then it gets saved successfully
+    And the new url is displayed
+    But the old url is not displayed
```


Wonderful done @alina-beck !!! 👍🏼
Love it! 😻

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-07-23T13:16:21Z" title="Tuesday, July 23rd 2019, 3:16:21 pm +02:00">Jul 23, 2019</time>_

---


Thanks. I really loved writing cypress tests, they're awesome! :grin: